front/src/pages/PlayPage.tsx

```diff
@@ -1,32 +1,94 @@
-import TinderCard from "react-tinder-card";
+import { createRef, useMemo, useRef, useState } from 'react';
+import TinderCard from 'react-tinder-card';
+import './PlayPage.css';
 
-const onSwipe = (direction: string) => {
-  console.log("You swiped: " + direction);
-};
+const db = [
+  {
+    text: 'TEXT',
+  },
+  {
+    text: 'MORE TEXT',
+  },
+  {
+    text: 'HERE TEXT',
+  },
+  {
+    text: 'STUFF HERE',
+  },
+  {
+    text: 'TESTING STUFF',
+  },
+];
 
-const onCardLeftScreen = () => {
-  console.log("card left the screen");
-};
-
-// TODO: Polish this. See https://github.com/3DJakob/react-tinder-card-demo/blob/master/src/examples/Advanced.js
 function PlayPage() {
+  const [currentIndex, setCurrentIndex] = useState(db.length - 1);
+  const currentIndexRef = useRef(currentIndex);
+
+  const swiped = (_direction: string, index: number) => {
+    updateCurrentIndex(index - 1);
+  };
+
+  const outOfFrame = (idx: number) => {
+    console.log(
+      `card with index ${idx} left the screen!`,
+      currentIndexRef.current,
+    );
+    currentIndexRef.current >= idx && childRefs[idx].current.restoreCard();
+  };
+
+  const swipe = async (dir: string) => {
+    if (canSwipe && currentIndex < db.length) {
+      await childRefs[currentIndex].current.swipe(dir); // Swipe the card!
+    }
+  };
+  const childRefs = useMemo(
+    () =>
+      Array(db.length)
+        .fill(0)
+        .map((_) => createRef<any>()),
+    [],
+  );
+
+  const updateCurrentIndex = (val: number) => {
+    setCurrentIndex(val);
+    currentIndexRef.current = val;
+  };
+
+  const canSwipe = currentIndex >= 0;
+
   return (
-    <TinderCard
-      onSwipe={onSwipe}
-      onCardLeftScreen={onCardLeftScreen}
-      preventSwipe={["down", "up"]}
-    >
-      <div
-        style={{
-          height: 500,
-          backgroundImage:
-            "url(https://basepaws.com/_next/image?url=https%3A%2F%2Fimages.ctfassets.net%2Fqj2yxv39d6ig%2F4XXFhU2WQtuORzIVqFvoay%2Fdda16ebb32a51db8a620aac1fb9ee5e7%2Fj45Rv.jpg%3Fw%3D755%26h%3D503%26q%3D80&w=1920&q=75)",
-        }}
-        className="card"
-      >
-        <h3>Hey</h3>
+    <div>
+      <div className='cardContainer'>
+        {db.map((card, index) => (
+          <TinderCard
+            onSwipe={(dir) => swiped(dir, index)}
+            onCardLeftScreen={() => outOfFrame(index)}
+            preventSwipe={['down', 'up']}
+            className='card'
+            ref={childRefs[index]}
+            key={index}
+          >
+            <div className='innerCard'>
+              {card.text}
+            </div>
+          </TinderCard>
+        ))}
+      </div>
+      <div className='buttons'>
```

Comment:
It's okay for now, but I think it's best if we use the MUI component so that we have more consistency in the app. Using Grid and Container instead of simple div, using the MUI Button... We can also custom the style with this. Have a look at MUI library !